README.md

```diff
@@ -60,6 +60,7 @@ it's quite easy with Homebrew.
 $ brew install neovim --HEAD
 ```
 
+To check if Neovim's floating window feature is available, try ```echo exists('*nvim_open_win')``` which should echo `1`.
```

Comment:
Could you add `:` to the head of command to show it is a Vim command explicitly?

```
:echo exists('*nvim_open_win')
```

Reply:
Great catch!